Source/CombatExtended/CombatExtended/Jobs/Utils/JobGiverUtils_Reload.cs

```diff
@@ -126,8 +126,26 @@ public static bool CanReload(Pawn pawn, Thing hopefullyTurret, bool forced = fal
 		private static Thing FindBestAmmo(Pawn pawn, Building_TurretGunCE reloadable)
 		{
 			//ThingFilter filter = refuelable.TryGetComp<CompRefuelable>().Props.fuelFilter;
-			var requestedAmmo = reloadable.CompAmmo.SelectedAmmo;
+			AmmoDef requestedAmmo = reloadable.CompAmmo.SelectedAmmo;
```

Comment:
Yes, this is my workaround for the siege raids using only one type of ammo and being "stuck" (whether the ammo system is on or off).
I should restrict this to non-player actions?
The alternative is to restrict siege mortar ammo to a single type of shell, but I feel this wouldn't be as fun :)

Reply:
@N7Huntsman Your call.

Reply:
While I don't have a problem with raiders firing whatever shells they can get their hands on, I don't see this as something we want colonists doing--has the potential for a lot of undesirable behavior. Madara and I discussed it, and he suggested possibly adding a faction check or something, so that it doesn't apply to the player?

Reply:
It makes sense.
I added a `!pawn.IsColonist` check, which I assume is the canonical way to do this?